docs/api/turn-by-turn/api-reference.md

```diff
@@ -105,6 +105,8 @@ These options are available for `auto`, `bus`, and `truck` costing methods.
 | `use_tolls` | This value indicates the willingness to take roads with tolls. This is a range of values between 0 and 1. Values near 0 attempt to avoid tolls and values near 1 will not attempt to avoid them. The default value is 0.5. Note that sometimes roads with tolls are required to complete a route so values of 0 are not guaranteed to avoid them entirely. |
 | `use_living_streets` | This value indicates the willingness to take living streets. This is a range of values between 0 and 1. Values near 0 attempt to avoid living streets and values near 1 will favor living streets. The default value is 0 for trucks, 0.1 for cars, buses, motor scooters and motorcycles. Note that sometimes living streets are required to complete a route so values of 0 are not guaranteed to avoid living streets entirely. |
 | `use_tracks` | This value indicates the willingness to take track roads. This is a range of values between 0 and 1. Values near 0 attempt to avoid tracks and values near 1 will favor tracks a little bit. The default value is 0 for autos, 0.5 for motor scooters and motorcycles. Note that sometimes tracks are required to complete a route so values of 0 are not guaranteed to avoid tracks entirely. |
+| `service_penalty` | A penalty applied for transition to generic service road. The default penalty is 0 trucks and 15 for cars, buses, motor scooters and motorcycles. |
```

Comment:
0 for trucks because of 'low_class_penalty' truck option which covers this case to a certain extent

Reply:
do we need this context (`low_class_penalty` for truck) in the doc itself?

Reply:
I think it is ok to leave out.  Should be noted that we are considering adding a `residential_penalty` and the `low_class_penalty` may get removed.  I think it is better to have a penalty supported by all modes vs just one or the other.